cli.ts

```diff
@@ -6,48 +6,60 @@ import { readFileSync, writeFileSync, existsSync } from 'fs';
 
 const packages: Package[] = JSON.parse(execSync(`lerna ls --json --loglevel silent`).toString());
-const packagesDirectory = dirname(packages[0].location);
-
-packages.forEach(({ name, location }) => {
-  exec('npm ls --depth 0 --json --silent', { cwd: location }, (_error, stdout, _stderror) => {
-    const dependencyNames = Object.keys(JSON.parse(stdout).dependencies || {});
-    const paths = dependencyNames.map(getPathForDependency).filter(x => !!x);
-
-    if (paths.length == 0) {
-      return log(`${name} - 0 dependencies found.`);
-    }
-
-    const configFilePath = join(location, tsConfigFileName);
-    const config: TsConfig = JSON.parse(readFileSync(configFilePath).toString());
-    const { references = [] } = config;
-    const existingPaths = references.map(r => r.path);
-
-    if (paths.sort().toString() === existingPaths.sort().toString()) {
-      log(`${name} - ${paths.length} dependencies found. All match project references in ${tsConfigFileName}.`);
-    } else if (args['--update']) {
-      warn(`*UPDATE* - ${name} - Some dependencies didn't match. Writing ${paths.length} to ${tsConfigFileName}.`);
-      const newJsonConfig = JSON.stringify({ ...config, references: paths.map(path => ({ path })) });
-      writeFileSync(configFilePath, format(newJsonConfig, { parser: 'json' }));
-    } else {
-      error(`**FAIL** - ${name} - Some dependencies didn't match.`);
-      error(`**FAIL** - ${name} - From npm packages: ${paths}`);
-      error(`**FAIL** - ${name} - From ${tsConfigFileName}: ${existingPaths}`);
-      process.exit(1);
-    }
+
+packages
+  .filter(({ location }) => {
+    // Ignore non-ts package
+    return existsSync(join(location, tsConfigFileName));
+  })
+  .forEach(({ name, location }) => {
+    exec('npm ls --depth 0 --json --silent', { cwd: location }, (_error, stdout, _stderror) => {
+      const dependencyNames = Object.keys(JSON.parse(stdout).dependencies || {});
+      const paths = dependencyNames.map(getPathForDependency).filter(x => !!x);
+
+      if (paths.length == 0) {
+        return log(`${name} - 0 dependencies found.`);
+      }
+
+      const configFilePath = join(location, tsConfigFileName);
+      const config: TsConfig = JSON.parse(readFileSync(configFilePath).toString());
+      const { references = [] } = config;
+      const existingPaths = references.map(r => r.path);
+
+      if (paths.sort().toString() === existingPaths.sort().toString()) {
+        log(`${name} - ${paths.length} dependencies found. All match project references in ${tsConfigFileName}.`);
+      } else if (args['--update']) {
+        warn(`*UPDATE* - ${name} - Some dependencies didn't match. Writing ${paths.length} to ${tsConfigFileName}.`);
+        const newJsonConfig = JSON.stringify({
+          ...config,
+          references: paths.map(referencePath => {
+            // Convert absolute path to relative path
+            return {
+              path: path.relative(location, referencePath)
```

Comment:
Example Project Reference:

`packages/@scope/pkgA`
->
`packages/@scope/pkgB`


Previously, `packages/@scope/pkgA/tsconfig.json` get wrong path like `{ "path": "../@scope/pkgB" }`.
This commit fix this `path` correctly.

```diff
- { "path": "../@scope/pkgB" }
+ { "path": "../pkgB" }
```